tests/test_input_validator.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from core_tic_tac_toe.board import Board
+from core_tic_tac_toe.input_validator import InputValidator
+
+
+class TestInputValidator:
+
+    @pytest.fixture
+    def input_validator(self):
+        return InputValidator()
+
+    @pytest.fixture
+    def board(self):
+        return Board()
+
+    def test_when_user_move_input_is_alphabetic_expect_false(self, input_validator, board):
+        result_1 = input_validator.is_move_valid("A", board)
+        result_2 = input_validator.is_move_valid("b", board)
+        result_3 = input_validator.is_move_valid("bY", board)
+
+        assert result_1 == False
+        assert result_2 == False
+        assert result_3 == False
```

Comment:
When the expression being tested is fairly simple, I tend to not use local variables. What do you think about how this reads? You could also experiment with swapping the expected (`False`) and actual values in the assertion.

```suggestion
        assert False == input_validator.is_move_valid("A", board)
        assert False == input_validator.is_move_valid("b", board)
        assert False == input_validator.is_move_valid("bY", board)
```